Sources/CollectionViewManager.swift

```diff
@@ -506,6 +510,78 @@ open class CollectionViewManager: NSObject {
         }
     }
 
+    /// Replaces cell items inside the specified section item, and then updates corresponding cells within section.
+    ///
+    /// - Parameters:
+    ///   - indexes: An array of locations, that contains indexes of cell items to replace inside specified section item
+    ///   - cellItems: An array of replacement cell items, which respond for cell configuration at specified index path
+    ///   - sectionItem: Section item within which cell items should be replaced
+    ///   - performUpdates: A Boolean value determines whether the performBatchUpdates is called.
+    ///   - configureAnimated: A Boolean value determines whether configuration of cell item performed animated.
+    /// Cell item isReplacementAnimationEnabled value should be false in that case.
+    ///   - completion: A closure that either specifies any additional actions which should be performed after replacing.
+    open func softUpdate(cellItemsAt indexes: [Int],
```

Comment:
- softUpdate and replace have so much common 
- softUpdate as default CVM update